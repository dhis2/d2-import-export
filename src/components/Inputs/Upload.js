import React from 'react'
import { File } from '../FinalFormComponents/File'

export const Upload = () => <File name="upload" dataTest="input-upload" />
