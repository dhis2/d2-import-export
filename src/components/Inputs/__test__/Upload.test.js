import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { Upload } from '../Upload'

jest.mock('react-final-form')

describe('Input component - Upload', () => {
    it('should render correctly', () => {
        const file = shallow(<Upload />)

        expect(toJson(file)).toMatchSnapshot()
    })
})
