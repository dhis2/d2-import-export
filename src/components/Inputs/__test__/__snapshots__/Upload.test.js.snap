// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component - Upload should render correctly 1`] = `
<Field>
  <File
    dataTest="input-upload"
    name="upload"
  />
</Field>
`;
